app/theme/client/imports/general/base_old.css

```diff
@@ -2849,6 +2849,8 @@ rc-old select,
 
 		opacity: 1;
 
+		word-break: break-word;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "word-break" to come before "opacity" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4730944)